src/MergeContext.js

```diff
@@ -454,6 +454,11 @@ class BranchPosition
     }
 }
 
+function checkLineLength(line, limit) {
```

Comment:
If possible, I would use the default value here instead of duplicating it in callers:

```suggestion
function checkLineLength(line, limit = 72) {
```

Otherwise, we would need to move `72` from callers into a named DefaultLineLengthMax constant (to use in callers).

Reply:
>  use the default value

Done.